docs/install/windows-release-channels.md

```diff
@@ -0,0 +1,320 @@
+
+```powershell
+# Check Netdata service status
+Get-Service netdata | Select-Object Name, Status, DisplayName
+
+# Check service executable path
+Get-WmiObject win32_service | Where-Object {$_.Name -eq 'netdata'} | Select-Object PathName
+```
+
+</details>
+
+## Troubleshooting Common Issues
+
+<details>
+<summary><strong>Issue: "Another version is already installed"</strong></summary><br/>
+
+**Symptoms**: MSI installer shows error about existing installation
+
+**Solution**: Use the `REINSTALL=ALL` parameter explicitly:
+
+```powershell
+msiexec /qn /i netdata-x64.msi REINSTALL=ALL
+```
+
+Alternatively, uninstall first, then reinstall:
+
+```powershell
+# Uninstall existing installation
+msiexec /qn /x netdata-x64.msi
+
+# Wait for uninstall to complete
+Start-Sleep -Seconds 10
+
+# Reinstall with new channel
+msiexec /qn /i netdata-x64.msi TOKEN="<YOUR_TOKEN>" ROOMS="<YOUR_ROOMS>"
+```
+
+</details>
+
+<br/>
+
+<details>
+<summary><strong>Issue: Cloud Connection Lost After Switch</strong></summary><br/>
+
+**Symptoms**: After switching channels, node doesn't appear in Netdata Cloud
+
+**Solution**: Reclaim the node:
+
+```powershell
+# Stop Netdata service
+Stop-Service netdata
+
+# Remove old cloud configuration
+Remove-Item "C:\Program Files\Netdata\var\lib\netdata\cloud.d\*" -Force
+
+# Reinstall with claim token
+msiexec /qn /i netdata-x64.msi TOKEN="<YOUR_TOKEN>" ROOMS="<YOUR_ROOMS>" REINSTALL=ALL
+
+# Start service
+Start-Service netdata
+```
+
+</details>
+
+<br/>
+
+<details>
+<summary><strong>Issue: Service Won't Start After Switch</strong></summary><br/>
+
+**Symptoms**: Netdata service fails to start after channel switch
+
+**Solution**: Check service status and perform clean reinstall:
+
+```powershell
+# Check service status
+Get-Service netdata
+
+# Check Windows Event Log for errors
+Get-EventLog -LogName Application -Source Netdata -Newest 20
+
+# Perform clean reinstall
+msiexec /qn /x netdata-x64.msi
+Remove-Item "C:\Program Files\Netdata" -Recurse -Force -ErrorAction SilentlyContinue
+Remove-Item "$env:PROGRAMDATA\Netdata" -Recurse -Force -ErrorAction SilentlyContinue
+msiexec /qn /i netdata-x64.msi TOKEN="<YOUR_TOKEN>" ROOMS="<YOUR_ROOMS>"
+```
+
+</details>
+
+## When to Choose Each Channel
+
+**Choose Stable Channel If:**
+- You're running production systems
+- You need predictable, well-tested releases
+- You prefer less frequent updates (major releases only)
+- You want maximum stability over cutting-edge features
+- You have strict change management processes
+
+**Choose Nightly Channel If:**
+- You're testing new features before production deployment
+- You want immediate bug fixes without waiting for releases
+- You're contributing to Netdata development and need latest code
+- You want to provide early feedback to the Netdata team
+
+:::tip
+
+**Update Frequency**: Stable releases occur every few weeks to months, while Nightly builds are updated daily with every commit to the master branch.
```

Comment:
Ideally, this whole section would become a subsection of `## Understanding Windows Installation Architecture`. I suggest add it after `### Release Channels Available`